README.md

```diff
@@ -307,7 +307,7 @@ These should match the relevant column type. Mapped variables are substituted in
 A `variables` map usually isn't needed for simple queries. The basic condition string should automatically get converted to a meaningful type, but when this fails 
 replacing tricky elements with a variable may help.
 
-### Remote registration
+### Remote registration [Experimental - OMERO Plus Only]
 
 For **OMERO Plus** installations which support TileDB as the OMERO.tables backend 
 it is possible to register tables in-place in a similar manner to in-place image 
```

Comment:
Can you also clarify the language here? Non-remote table data is stored in OMERO's binary repository, but not within `ManagedRepository`.